Source/WebKit/UIProcess/Launcher/glib/BubblewrapLauncher.cpp

```diff
@@ -172,13 +172,16 @@ enum class BindFlags {
     Device,
 };
 
-static void bindSymlinksRealPath(Vector<CString>& args, const String& path, const char* bindOption = "--ro-bind")
+static bool bindSymlinksRealPath(Vector<CString>& args, const String& path, const char* bindOption = "--ro-bind")
 {
     auto realPath = FileSystem::realPath(path);
     if (path != realPath) {
         CString rpath = realPath.utf8();
         args.appendVector(Vector<CString>({ bindOption, rpath.data(), rpath.data() }));
+        return true;
     }
+
+    return false;
```

Comment:
I expect this change will cause breakage.

Previously what it did was:

1. Find where a symlink pointed, and mount where it points
2. Continued on to mount the symlink path itself

Now it will be broken if the symlink is in a path not already mounted.

Reply:
I'll revert this then :)